test/commons/dom/get-composed-parent.js

```diff
@@ -0,0 +1,72 @@
+/* global xit */
+describe('dom.getComposedParent', function () {
+  'use strict';
+  var getComposedParent = axe.commons.dom.getComposedParent;
+  var fixture = document.getElementById('fixture');
+  var shadowSupport = document.body && typeof document.body.attachShadow === 'function';
```

Comment:
The fact that there are different APIs for v0 and v1 make SD pretty difficult to work with. I'd be curious to hear how much test coverage we need for both.